indico/modules/rb/client/js/common/rooms/RoomEditModal.module.scss

```diff
@@ -13,11 +13,15 @@
 }
 
 .submit-message {
-    margin-bottom: 10px;
+    margin: 0 0 1em !important;
 }
 
 .room-form {
     :global(.fields) > :global(.field) {
         flex-grow: 1 !important;
     }
 }
+
+.DateRangePicker_picker { // sass-lint:disable-line class-name-format
```

Comment:
don't you need `:global()` here?

Reply:
Works fine with both approaches. I'm keeping `:global()`.